check/Cargo.toml

```diff
@@ -25,6 +25,8 @@ gluon_parser = { path = "../parser", version = "0.5.0", optional = true } # GLUO
 collect-mac = "0.1.0"
 pretty_assertions = "0.3.4"
 
+gluon_completion = { path = "../completion", version = "0.5.0" } # GLUON
```

Comment:
I completely agree, I just wasn't sure on your preference on what to do with check/tests/support/mod.rs. If more tests are added and more duplication is needed, it could be split into a separate test-harness crate, but for now I think the duplication is a reasonable solution. I'll make the change.